modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/climax.dm

```diff
@@ -185,8 +197,13 @@
 							span_userlove("You hilt your cock into [target_human]'s [climax_into_choice], shooting cum into [target_human_them]!"))
 						to_chat(target_human, span_userlove("Your [climax_into_choice] fills with warm cum as [src] shoots [self_their] load into it."))
 
-			var/obj/item/organ/genital/testicles/testicles = get_organ_slot(ORGAN_SLOT_TESTICLES)
-			testicles.transfer_internal_fluid(null, testicles.internal_fluid_count * 0.6) // yep. we are sending semen to nullspace
+					if(climax_into_choice == "mouth")
```

Comment:
this can be added to the `else` above it instead of being separate out here i.e. indent it so it runs after the last to_chat in there, that's the only time it makes sense to check it